src/panels/config/lovelace/dashboards/ha-config-lovelace-dashboards.ts

```diff
@@ -164,6 +161,7 @@ export class HaConfigLovelaceDashboards extends LitElement {
         .columns=${this._columns(this.hass.language, this._dashboards)}
         .data=${this._getItems(this._dashboards)}
         @row-click=${this._editDashboard}
+        id="url_path"
```

Comment:
Is this used ?

Reply:
Yes, since yaml dashboards don't have an id, and the url_path is unique, I use it to identify the clicked row.